_docs/setup/install-kubernetes.md

```diff
@@ -231,52 +199,34 @@ kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
 
 ## Uninstalling
 
-You can use the [Istio Helm chart](https://github.com/kubernetes/charts/tree/master/incubator/istio#uninstalling-the-chart) to uninstall, or follow the steps below.
-
-1. Uninstall Istio core components:
-
-   * If Istio was installed without Istio auth feature:
+Upgrade is not yet supported. Please un-install using the same yaml files you used during install.
 
-     ```bash
-     kubectl delete -f install/kubernetes/istio.yaml
-     ```
+1. If you installed Istio addons, uninstall them:
 
-   * If Istio was installed with auth feature enabled:
+   ```bash
+   kubectl delete -f install/kubernetes/addons/
+   ```
 
-     ```bash
-     kubectl delete -f install/kubernetes/istio-auth.yaml
-     ```
 
-2. Uninstall RBAC Istio roles:
+1. Uninstall Istio core components. For the {{ site.data.istio.version }} release, the uninstall will delete the RBAC permissions, the `istio-system` namespace, and hierarchically all resources under it.
+   It is safe to ignore errors for non-existent resources, since they may have been deleted hierarchically.
 
-   * If beta version was installed:
+   * If Istio was installed without mTLS authentication enabled:
 
      ```bash
-     kubectl delete -f install/kubernetes/istio-rbac-beta.yaml
+     kubectl delete -f install/kubernetes/istio.yaml
      ```
 
-   * If alpha version was installed:
+   * If Istio was installed with mTLS authentication enabled:
 
      ```bash
-     kubectl delete -f install/kubernetes/istio-rbac-alpha.yaml
+     kubectl delete -f install/kubernetes/istio-auth.yaml
      ```
 
-3. If you installed Istio addons, uninstall them:
-
-   ```bash
-   kubectl delete -f install/kubernetes/addons/
-   ```
-   and, if Jaeger was installed,
-   ```bash
-   kubectl delete -f https://raw.githubusercontent.com/jaegertracing/jaeger-kubernetes/master/all-in-one/jaeger-all-in-one-template.yml
-   ```
-
-
-4. Delete Istio Kubernetes [TPRs](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-third-party-resource):
+1. Delete Istio's [Kubernetes CRDs](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/):
 
    ```bash
-   kubectl delete istioconfigs --all
-   kubectl delete thirdpartyresource istio-config.istio.io
+   kubectl delete crd --all
```

Comment:
Wow, this is dangerous isn't it? We might delete somebody else's CRDs as well. Isn't there a scoped command for this?